src/Orc.Automation/ScenarioManagement/ScenarioManager.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace Orc.Automation.ScenarioManagement;
+
+using System;
+
+public static class ScenarioManager
+{
+    private static AutomationScenario? _currentScenario;
+
+    public static AutomationScenario? CurrentScenario => _currentScenario;
+
+    public static bool IsRecordingStepsSuspended { get; set; }
+
+    public static void StartScenario(AutomationScenario scenario)
```

Comment:
Should we do async by design?

Reply:
maybe... it so rare stuff (I expect in a future it would be more usages) ...maybe for next PR?